packages/ketcher-react/src/script/editor/tool/rgroupatom.js

```diff
@@ -35,7 +35,14 @@ function RGroupAtomTool(editor) {
 }
 
 RGroupAtomTool.prototype.mousemove = function (event) {
-  this.editor.hover(this.editor.findItem(event, ['atoms']))
+  const struct = this.editor.render.ctab.molecule
```

Comment:
it might be nice to move this part into common util function, and reuse it, if possible